envConfig/live.env

```
@@ -8,3 +8,4 @@ SIMORGH_TOGGLES_URL=https://toggles.api.bbci.co.uk
 LOG_DIR='/var/log/simorgh'
 LOG_LEVEL='info'
 SIMORGH_APP_ENV=live
+SIMORGH_INCLUDES_ENDPOINT=https://bbc.com/ws/includes
```

```suggestion
SIMORGH_INCLUDES_ENDPOINT=https://www.bbc.com/ws/includes
```
You may want to change this because https://bbc.com/ws/includes returns a 301 code and redirects to www.bbc.com, this may make includes load slightly slower

---

thanks @andrewscfc